commit 7fe0e29ab432a4ce0753f287db4efcd8a441c6e1 (HEAD, REL9_3_STABLE)
Author:     Pavel Raiskup <praiskup@redhat.com>
AuthorDate: Mon Jul 14 09:00:03 2014 +0200
Commit:     Pavel Raiskup <praiskup@redhat.com>
CommitDate: Mon Jul 14 09:00:03 2014 +0200

    Revert "backpatch fix to build to allow travis CI to build it"
    
    This reverts commit 88b9a034c8dd86a1de961f370e548cac74ed1031.

diff --git a/build.xml b/build.xml
index 152517d..2b86a36 100644
--- a/build.xml
+++ b/build.xml
@@ -106,10 +106,6 @@
     <include name="${package}/xa/jdbc4/*.java" if="jdbc4any"/>
   </patternset>
 
-  <artifact:dependencies pathId="dependency.classpath" useScope="test">
-    <dependency groupId="junit" artifactId="junit" version="3.8.2" scope="test"/>
-  </artifact:dependencies>
-
   <target name="check_versions">
     <condition property="jdbc2">
       <or>
@@ -233,7 +229,6 @@
     <fail message="Old driver was detected on classpath or in jre/lib/ext, please remove and try again." if="old.driver.present" />
 
     <javac classpath="${srcdir}" srcdir="${srcdir}" destdir="${builddir}" debug="${debug}" source="${java.specification.version}">
-      <classpath refid="dependency.classpath" />
       <patternset refid="jdbc.version.src.pattern"/>
     </javac>
   </target>
@@ -434,9 +429,9 @@
 
   <!-- This compiles and builds the test jarfile. -->  
   <target name="testjar" depends="jar">
+    <fail message="JUnit could not be found in your classpath.  You must download and install it from http://junit.org to build and run the test suite." unless="junit" />
     <mkdir dir="${builddir}/tests"/>
     <javac srcdir="${srcdir}" destdir="${builddir}/tests" debug="${debug}" source="${java.specification.version}">
-      <classpath refid="dependency.classpath" />
       <include name="${package}/test/**" />
 
       <exclude name="${package}/test/jdbc4/**" unless="jdbc4tests" />
@@ -472,7 +467,6 @@
       <sysproperty key="protocolVersion" value="${protocolVersion}" />
       <sysproperty key="ssltest.properties" value="${ssltest.properties}" />
 
-      <classpath refid="dependency.classpath" />
       <classpath>
         <pathelement location="${artifact.jar}" />
         <pathelement location="${jardir}/postgresql-tests.jar" />
